index.html

```diff
@@ -651,35 +651,128 @@ <h4>Registration Information</h4>
                                 <td><code>created</code></td>
-                                <td>Provides information when the TD instance was
-                                    created inside the directory.</td>
-                                <td>system-generated (read-only)</td>
+                                <td>
+                                    Provides information when the TD instance was
+                                    created inside the directory.
+                                    <p>
+                                        This MAY be set by the directory and returned to consumers.
+                                    </p>
+                                </td>
+                                <td>read-only</td>
+                                <td>optional</td>
                                 <td><a href="http://www.w3.org/TR/2012/REC-xmlschema11-2-20120405/#dateTime"><code>dateTime</code></a></td>
                             </tr>
-                            <tr class="rfc2119-table-assertion" id="tdd-vocab-modified--Thing">
+                            <tr class="rfc2119-table-assertion" id="discovery-vocab-modified--RegistrationInformation">
                                 <td><code>modified</code></td>
-                                <td>Provides information when the TD instance was
-                                    last modified inside the directory.</td>
-                                <td>system-generated (read-only)</td>
+                                <td>
+                                    Provides information when the TD instance was
+                                    last modified inside the directory.
+                                    <p>
+                                        This MAY be set by the directory and returned to consumers.
+                                    </p>
+                                </td>
+                                <td>read-only</td>
+                                <td>optional</td>
                                 <td><a href="http://www.w3.org/TR/2012/REC-xmlschema11-2-20120405/#dateTime"><code>dateTime</code></a></td>
                             </tr>
-                            <!-- <tr class="rfc2119-table-assertion" id="tdd-vocab-expires--Thing">
+                            <tr class="rfc2119-table-assertion" id="discovery-vocab-expires--RegistrationInformation">
                                 <td><code>expires</code></td>
-                                <td>Provides information when the TD instance registration will
-                                    expire.</td>
+                                <td>
+                                    Provides the time for when the TD instance registration expires.
+                                    <p>
+                                        The producer MAY set this to indicate the absolute expiry time
```

Comment:
be careful to avoid implying that there is a different data type for absolute and relative expiry time.  Rather than using these terms I would explicitly refer to "ttl" and "expiry" (e.g. say if one is present, the other should not be, etc).